src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -339,7 +339,6 @@ class GoogleCloudPlatformProvider(schema.Base):
     @model_validator(mode="before")
     @classmethod
     def _check_input(cls, data: Any) -> Any:
-        google_cloud.check_credentials()
```

Comment:
This is not needed anymore as the `regions()` function below will call it under the hood.